arangodb-net-standard/Transport/Http/HttpApiTransport.cs

```diff
@@ -78,33 +95,61 @@ public HttpApiTransport(HttpClient client)
         /// <param name="hostUri"></param>
         /// <param name="dbName"></param>
         /// <param name="jwtToken"></param>
+        /// <param name="contentType">Content type to use in requests.
+        /// Used to set Content-Type and Accept HTTP headers.
+        /// Uses JSON content type by default.</param>
         /// <returns></returns>
         public static HttpApiTransport UsingJwtAuth(
             Uri hostUri,
             string dbName,
-            string jwtToken)
+            string jwtToken,
+            HttpContentType contentType = HttpContentType.Json)
         {
             var client = new HttpClient();
             client.BaseAddress = new Uri(hostUri.AbsoluteUri + "_db/" + dbName + "/");
 
-            var transport = new HttpApiTransport(client);
+            var transport = new HttpApiTransport(client, contentType);
             transport.SetJwtToken(jwtToken);
 
             return transport;
         }
 
+        /// <summary>
+        /// Make this <see cref="HttpApiTransport"/> instance use JSON content type
+        /// for Content-Type and Accept HTTP headers.
+        /// </summary>
+        public void UseJsonContentType()
+        {
+            _contentType = HttpContentType.Json;
+            _client.DefaultRequestHeaders.Accept.Clear();
+            _client.DefaultRequestHeaders.Accept.Add(
+                new MediaTypeWithQualityHeaderValue(_contentTypeMap[_contentType]));
+        }
+
+        /// <summary>
+        /// Make this <see cref="HttpApiTransport"/> instance use VPack content type
+        /// for Content-Type and Accept HTTP headers.
+        /// </summary>
+        public void UseVPackContentType()
```

Comment:
Should we document that the `Accept` header of the underlying `HttpClient` gets cleared? I wonder if some users could provide a client that is setup to use different kind of content at once.

Reply:
Meh... Is that really likely? Sounds like user error to me... There shouldn't be more than one, this is specifically for interacting with ArangoDB.

Reply:
Yeah fair enough. I've approved the PR 👍